lib/components/artist/shows/__tests__/show-tests.js

```diff
@@ -0,0 +1,41 @@
+import 'react-native'
+
+import React from 'react'
+import renderer from 'react-test-renderer'
+
+import Show from '../show'
+jest.mock('../../../opaque_image_view.js', () => 'AROpaqueImageView')
+it('renders properly', () => {
+  const showProps = {
+    href: 'artsy.net/show',
+    cover_image: {
+      url: 'artsy.net/image-url'
+    },
+    kind: 'solo',
+    name: 'Expansive Exhibition',
+    exhibition_period: 'Jan 1 - March 1',
+    status_update: 'Closing in 2 days',
+    status: 'running',
+    partner: {
+      name: 'Gallery'
+    },
+    location: {
+      city: 'Berlin'
+    }
+  }
+
+  const showStyles = {
```

Comment:
It normally comes in from it's parent component ([VariableSizeShowsList](https://github.com/artsy/emission/blob/master/lib/components/artist/shows/variable_size_shows_list.js#L62])) and has no defaults to fall back to.

This test will implicitly validate that any style passed in via props would be re-applied on the component